docs/ml/deploy/_index.md

```diff
@@ -155,11 +155,11 @@ output_tensors = await my_mlmodel.infer(input_tensors)
 For more information, see the [Go SDK Docs](https://pkg.go.dev/go.viam.com/rdk/services/mlmodel#Service).
 
 ```go {class="line-numbers linkable-line-numbers"}
-myMLModel, err := mlmodel.FromRobot(robot, "my_mlmodel_service")
+myMLModel, err := mlmodel.FromRobot(machine, "my_mlmodel_service")
 
 input_tensors := ml.Tensors{"0": tensor.New(tensor.WithShape(1, 2, 3), tensor.WithBacking(6))}
 
-output_tensors, err := myMLModel.Infer(ctx.Background(), input_tensors)
+output_tensors, err := myMLModel.Infer(context.Background(), input_tensors)
```

Comment:
Mentioned this offline, and I realize this might a docs-wide change (so feel free to ignore), but ideally example Go code that uses `context.Background()` should really use [`context.TODO()`](https://pkg.go.dev/context#TODO) instead.

Reply:
For sure!